gson/src/test/java/com/google/gson/JsonArrayTest.java

```diff
@@ -20,6 +20,9 @@
 
 import com.google.gson.common.MoreAsserts;
 
+import static org.junit.Assert.*;
```

Comment:
It's not clear to me why these imports are needed. Aren't `assertEquals` etc inherited from `TestCase`?

Reply:
I have removed those imports.